test/miscellaneous_tests/units.jl

```diff
@@ -1,103 +1,191 @@
-#! format: off
+### Prepares Tests ###
 
-using Catalyst, Unitful, Test
-const MT = ModelingToolkit
+# Fetch packages.
+using Catalyst, DynamicQuantities, Test
+using ModelingToolkit: get_iv, get_unit, validate, ValidationError
 
-### Run Tests ###
 
+### Basic Tests ###
+
+# Checks that units work with programmatic model creation.
 let
```

Comment:
The new test is a rewriting of the old, unfortunately it seems there was some many changes that Git just makes it all as changed.